src/app.py

```diff
@@ -122,6 +110,10 @@ class getObject(Resource):
     @ns.doc("GetObject")
     @api.response(200, "Success", model=GetObject_example)
     def get(self):
+        # TODO: make into a DAO layer with transactions
+        with open("../keys.json", "r") as f:
```

Comment:
I moved the open/write for keys.json into each endpoint's method so when we have a database, it will be easier to model everything as transactions. This way, the "database" is written to after every PutObject rather than once after the app closes.